packages/terra-list/src/ListSubsection.jsx

```diff
@@ -56,33 +73,125 @@ const ListSubsection = ({
   children,
   isCollapsed,
   isCollapsible,
+  isDraggable,
+  onDragEnd,
+  intl,
   ...customProps
 }) => {
-  let sectionItems;
-  if (!isCollapsible || !isCollapsed) {
-    sectionItems = children;
-  }
+  const [listItemNodes, setlistItemNodes] = useState(children);
+  let listNode = useRef();
+
+  useEffect(() => {
+    if (!isCollapsible || !isCollapsed) {
```

Comment:
Same concern as above.

Reply:
Yes. This is the existing logic in terra-list component. Kept intact. Thanks